client/state/i18n/locale-suggestions/actions.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { I18N_LOCALE_SUGGESTIONS_ADD, I18N_LOCALE_SUGGESTIONS_REQUEST } from 'state/action-types';
+
+/**
+ * Action creator function: I18N_LOCALE_SUGGESTIONS_ADD
+ *
+ * @param {Object} items - list of locale suggestions
+ * @return {Object} action object
+ */
+export const receiveLocaleSuggestions = items => ( {
+	type: I18N_LOCALE_SUGGESTIONS_ADD,
+	items,
+} );
+
+/**
+ * Action creator to request locale suggestions: I18N_LOCALE_SUGGESTIONS_ADD
+ *
+ * @return {Object} action object
+ */
+export const requestLocaleSuggestions = () => {
+	return {
+		type: I18N_LOCALE_SUGGESTIONS_REQUEST,
+	};
+};
```

Comment:
fun fact! you can do as above in `receiveLocaleSuggestions` and use the object-return shorthand syntax…

```js
export const requestingLocaleSuggestions = () => ( {
	type: I18N_LOCALE_SUGGESTIONS_REQUEST
} )
```